android/source/VideoLocker/androidTest/java/org/edx/mobile/test/DateParseTests.java

```diff
@@ -58,4 +58,17 @@ public void testEmail() throws Exception {
         print("sending email...");
     }
 
+    /**
+     * This tests the formatted start date of a course
+     * @throws Exception
+     */
+    public void testFormatCourseNotStartedDate() throws Exception {
```

Comment:
This test will fail in non-english locales. Having a test that only works for some developers is pretty confusing.